enable/markers.py

```diff
@@ -171,6 +171,87 @@ def _add_to_path(self, path, size):
                           (0, -0.732 * size))))
 
 
+class LeftTriangleMarker(AbstractMarker):
+    """ A marker that is a triangle with one apex pointing left.
+    """
+    # How this marker is to be stroked. (Overrides AbstractMarker.)
+    draw_mode = FILL_STROKE
+    # Do not render anti-aliased. (Overrides AbstractMarker.)
+    antialias = False
```

Comment:
I don't think that we want this: the markers that have this set have corresponding Kiva versions implemented by Agg in C++ without aliasing, so this is to give pixel-to-pixel similarity between the two ways of drawing between different backends.  For these, we have no corresponding Agg versions (unless you want to expose the ones here: https://github.com/enthought/enable/blob/dddfac16132051e40d93485333f1c13ea7964631/kiva/agg/agg-24/include/agg_renderer_markers.h by adding them here: https://github.com/enthought/enable/blob/dddfac16132051e40d93485333f1c13ea7964631/kiva/agg/src/constants.i#L57 and possibly opengl implementations here: https://github.com/enthought/enable/blob/dddfac16132051e40d93485333f1c13ea7964631/kiva/agg/src/gl_graphics_context.cpp#L63 ...)

They will draw slightly slower, but look much nicer if antialias is True.